### PR TITLE
Phase 1 - 1.2 Use MySQL

### PR DESCRIPTION
**Pull request by kanghowoo**
Let’s implement storage implementation using MySQL.
And replace the storage implementation used in the service layer to this new storage implementation.
Check your application API for WRITE post working well with MySQL and create PR

In this project, using mybatis.

### CODE REVIEW

File: src/main/java/com/mide/gangsaeng/board/Board.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.mide.gangsaeng.board;
+
+public class Board {
+    private int id;
+    private String title;
+    private String content;
+    private int user_id;
```
Comment (TK-one) [inline src/main/java/com/mide/gangsaeng/board/Board.java:7]:
```suggestion
    private int userId;
```

File: src/main/java/com/mide/gangsaeng/board/BoardRequest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class BoardRequest {
+    private String title;
+    private String content;
+    private int user_id;
```
Comment (TK-one) [inline src/main/java/com/mide/gangsaeng/board/BoardRequest.java:11]:
```suggestion
    private int userId;
```

File: src/main/java/com/mide/gangsaeng/board/Board.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.mide.gangsaeng.board;
+
+public class Board {
```
Comment (TK-one) [inline src/main/java/com/mide/gangsaeng/board/Board.java:3]: Change to Immutable?

**Review by TK-one — APPROVED** [review]

File: src/main/java/com/mide/gangsaeng/board/Board.java (hunk 3 above)
Comment (kanghowoo) [reply src/main/java/com/mide/gangsaeng/board/Board.java:8]: Yes, VO (value object) is Read-Only , so make immutable is better choice.

**Pull request merged**